src/common/redis_module/ray_redis_module.cc

```diff
@@ -431,6 +472,68 @@ int TableLookup_RedisCommand(RedisModuleCtx *ctx,
   return REDISMODULE_OK;
 }
 
+bool is_nil(const std::string &data) {
+  assert(data.size() == 20);
```

Comment:
CHECK?  To be consistent elsewhere in the codebase

Reply:
Changed it